Jenkinsfile

```diff
@@ -28,7 +28,7 @@ pipeline {
       agent any
       steps {
         sh "kubectl apply --record -f kubernetes/"
```

Comment:
This line will need the azure context too.

Reply:
and that's why we review PRs!